<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basetab</class>
 <widget class="BaseTab" name="basetab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainTab</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="databaseView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Aktionen</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="layoutAction">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="newEntry">
          <property name="text">
           <string>Neu (N)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editEntry">
          <property name="text">
           <string>Bearbeiten (B)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteEntry">
          <property name="text">
           <string>Löschen (L)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="search">
          <property name="text">
           <string>Suchen (U)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter">
          <property name="text">
           <string>Ansicht (A)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pdfExport">
          <property name="text">
           <string>PDF Export (P)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="printEntry">
          <property name="text">
           <string>Drucken (D)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="goBack">
          <property name="text">
           <string>Zurück (ESC)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>201</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseTab</class>
   <extends>QWidget</extends>
   <header>tabs/basetab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>newEntry</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>AddEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteEntry</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>DeleteEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>SearchEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filter</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>FilterList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEntry</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>EditEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pdfExport</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>ExportToPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printEntry</sender>
   <signal>clicked()</signal>
   <receiver>basetab</receiver>
   <slot>PrintEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddEntry()</slot>
  <slot>DeleteEntry()</slot>
  <slot>SearchEntry()</slot>
  <slot>FilterList()</slot>
  <slot>EditEntry()</slot>
  <slot>PrintEntry()</slot>
  <slot>ExportToPDF()</slot>
  <slot>GoBack()</slot>
 </slots>
</ui>
